openlane/user_project_wrapper/fixed_dont_change/default_wrapper_cfgs.tcl

```diff
@@ -15,10 +15,14 @@
 
 # THE FOLLOWING SECTIONS CAN BE CHANGED IF NEEDED
 
-# PDN Pitch
-set ::env(FP_PDN_VPITCH) 180
-set ::env(FP_PDN_HPITCH) $::env(FP_PDN_VPITCH)
+# PDN Horizontal Pitch as mutliples of 30. Horizontal Pitch = 60 + FP_PDN_HPITCH_MULT * 30. 
+# FP_PDN_HPITCH_MULT is an integer. Minimum value is 0.
+set ::env(FP_PDN_HPITCH_MULT) 1
```

Comment:
If a user is allowed to change this value, it should be in the main config.tcl file

Reply:
User is allowed to change anything in the default wrapper config as well